builder/builder.go

```diff
@@ -113,19 +113,20 @@ func (b *Builder) ReadBuilderConf() {
 
 	// Map the builder values to the regex here to make it easier to assign
 	fields := []struct {
-		re   string
-		dest *string
+		re       string
+		dest     *string
+		required bool
 	}{
-		{`^BUNDLE_DIR\s*=\s*`, &b.Bundledir},
-		{`^CERT\s*=\s*`, &b.Cert},
-		{`^CLEARVER\s*=\s*`, &b.Clearver},
-		{`^FORMAT\s*=\s*`, &b.Format},
-		{`^MIXVER\s*=\s*`, &b.Mixver},
-		{`^REPODIR\s*=\s*`, &b.Repodir},
-		{`^RPMDIR\s*=\s*`, &b.Rpmdir},
-		{`^SERVER_STATE_DIR\s*=\s*`, &b.Statedir},
-		{`^VERSIONS_PATH\s*=\s*`, &b.Versiondir},
-		{`^YUM_CONF\s*=\s*`, &b.Yumconf},
+		{`^BUNDLE_DIR\s*=\s*`, &b.Bundledir, true},
+		{`^CERT\s*=\s*`, &b.Cert, true},
+		{`^CLEARVER\s*=\s*`, &b.Clearver, false},
```

Comment:
Are you guys using this variable in the conf? Or the MIXVER one?

Reply:
We aren't using them directly in our embargoed tooling.
The ones I marked as 'required' were based on my guess and what the mixer code appears to rely upon.

Reply:
Ok, I think those aren't use anywhere at this point, so I'll remove them in a different patch. The rest look correct to me though!